src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = 40;
+  const numberOfDays = days;
```

Comment:
The variable 'numberOfDays' is redundant since it's a direct assignment from the 'days' parameter. You can use 'days' directly in your calculations.